src/schedule/2021-02-10-using-storybook-with-redwood.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Using Storybook with Redwood
+type: Talk
+date: 2021-02-10T10:00:00-08:00
+---
```

Comment:
There's a `speakers` property we should apply here. The property is _thoroughly_ undocumented, which is a gap we should remedy sometime.

```suggestion
speakers:
  - Anthony Campolo
---
```